src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -174,9 +177,13 @@ trait WorkspaceApiService extends FireCloudRequestBuilding with FireCloudDirecti
                 // GET importPFB is deprecated; use GET importJob instead
                 path(("importPFB" | "importJob")) {
                   get {
-                    requireUserInfo() { _ =>
-                      extract(_.request.uri.query()) { query =>
-                        passthrough(Uri(encodeUri(s"${FireCloudConfig.ImportService.server}/$workspaceNamespace/$workspaceName/imports")).withQuery(query), HttpMethods.GET)
+                    requireUserInfo() { userInfo =>
+                      parameter(Symbol("running_only").as[Boolean].withDefault(false)) { runningOnly =>
+                        complete {
+                          entityServiceConstructor(FlexibleModelSchema).listJobs(workspaceNamespace, workspaceName, runningOnly, userInfo) map { respBody =>
+                            RequestComplete(OK, respBody)
+                          }
+                        }
```

Comment:
this replaces the previous passthrough (i.e. proxy) implementation for listing jobs. Now, we make the REST call, deserialize the results, then reply with those results.